Tests/SentryTests/Integrations/Performance/IO/SentryNSDataTrackerTests.swift

```diff
@@ -22,12 +23,14 @@ class SentryNSDataTrackerTests: XCTestCase {
     override func setUp() {
         super.setUp()
         fixture = Fixture()
-        fixture.getSut().enable()
+        fixture.getSut().enable(with: Options())
         SentrySDK.start { $0.enableFileIOTracing = true }
     }
     
     override func tearDown() {
         super.tearDown()
+        SentryNSDataTracker.sharedInstance.disable()
+        Dynamic(SentryNSDataTracker.sharedInstance).processInfoWrapper = SentryProcessInfoWrapper()
```

Comment:
`m`: That's a testing smell. We should remove the `sharedInstance` and move the singleton to the `SentryDependencyContainer`, so we don't have to clean up global state here.

Reply:
Discussion about singleton or not singleton in here https://github.com/getsentry/sentry-cocoa/pull/2573/files#r1071053357